setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
```

Comment:
I think we can even get rid of this `setup.cfg` and move everything inside the `pyproject.toml`. I can do this in a separate PR.